install/kubernetes/cilium/values.yaml

```diff
@@ -104,6 +104,12 @@ tolerations:
   #   value: "value"
   #   effect: "NoSchedule|PreferNoSchedule|NoExecute(1.6 only)"
 
+# -- Host aliases for cilium-agent.
+hostAliases: []
+# - ip: 1.1.1.1
+#   hostnames:
+#   - cloudflare.dns
```

Comment:
Can you add in the commit message why this is being added?

Reply:
@aanm This is because of cluster-mesh.
Cluster-mesh API server has **fixed** domain at [`<cluster-name>.mesh.cilium.io`](https://github.com/cilium/cilium/blob/2a46fd67267716efd58311064e4a91abfa2a0a1c/install/kubernetes/cilium/templates/_helpers.tpl#L92). Because it's cilium.io sub-domain, we have no way to config global DNS except using `hostAliases` as [`cilium` CLI does](https://docs.cilium.io/en/v1.10/gettingstarted/clustermesh/clustermesh/#connect-clusters).

Reply:
Sorry, if I take an example that let's u confuse.

Reply:
@aanm P/S, example updated:
```yaml
# -- Host aliases for cilium-agent.
hostAliases: []
# - ip: 10.10.xx.xx
#   hostnames:
#   - cluster1.mesh.cilium.io
```